lib/index.js

```diff
@@ -113,7 +113,7 @@ const combinePRs = async (
     );
   }
 
-  await execa("git", ["push", "origin", combineBranchName]);
+  await execa("git", ["push", "origin", combineBranchName, "-f"]);
```

Comment:
I can't say I'm a huge fan of this. We should be creating a new branch; if the branch exists on the remote, I don't think we should overwrite it, I think we should fail. So I don't want to include this here.

Reply:
It makes sense and I agree with you. I added it because (with no branch in the remote) I was having a fast-forward error on the git push. I'm going to test a better way to solve that problem. Maybe adding a new option. I will let you know! :)

Reply:
I will create an action to merge and delete the branch :) I should have solved this comment now